ports/geckolib/wrapper.rs

```diff
@@ -108,13 +107,9 @@ impl TRestyleDamage for GeckoRestyleDamage {
 
     fn compute(source: &nsStyleContext,
                new_style: &Arc<ComputedValues>) -> Self {
-        type Helpers = ArcHelpers<ServoComputedValues, ComputedValues>;
         let context = source as *const nsStyleContext as *mut nsStyleContext;
-
-        Helpers::borrow(new_style, |new_style| {
-            let hint = unsafe { Gecko_CalcStyleDifference(context, new_style) };
-            GeckoRestyleDamage(hint)
-        })
+        let hint = unsafe { Gecko_CalcStyleDifference(context, ComputedValues::to_borrowed(new_style)) };
```

Comment:
Hm... should we implement the trait as methods in Arc<T>, instead of static methods? (And giving them more meaningful names?)

I'd find way more readable and readable at a first look something like:

```rust
Gecko_CalcStyleDifference(context, new_style.as_borrowed_gecko_type())
```

(Well, maybe that name is too long, but you get the idea).